### ISSUE TITLE
Can't create ipad 5th gen ios16.1

### ISSUE THREAD

**Issue by ranzhie07** [verygenericname/SSHRD_Script#119]
![image](https://user-images.githubusercontent.com/13744581/201042192-9464d92c-c537-45dc-a6f8-9769268f1713.jpeg)

**Comment by ICH525** [comment]
YUP SAME PROBLEM AS ME happening right now.. any solutions you got?

**Comment by ScknCbt** [comment]
have some one found a solution for this.

**Comment by mast3rz3ro** [comment]
Try using [SSHRD Lite](https://github.com/mast3rz3ro/SSHRD_Script_Lite/)  it's supports decrypting iboot files directly with img4
